backend/app/db/crud/document.py

```diff
@@ -549,12 +543,15 @@ def write_metadata(
         db.add(doc_keyword)
 
 
-def _get_associated_documents(
+def _get_related_documents(
     db: Session, document_id: int
 ) -> Set[DocumentOverviewResponse]:
-    # Get all associated documents
-    # start with all documents with a reference from this document_id
-    related_documents_to = set(
+    """Gets all the other documents this document is related to."""
+    query_all_relationships_of_document = db.query(
+        DocumentRelationship.relationship_id
+    ).filter(DocumentRelationship.document_id == document_id)
+
+    return set(
```

Comment:
as we talked about separately, it would be good at some point in the future to return these as grouped sets of related documents with details about the relationship. That would require a FE update to support, and doesn't really matter at the moment while we only ever have a single relationship for a group of docs anyway! @PatFawbertMills we'll raise a ticket to keep track of this.

Reply:
Done https://linear.app/climate-policy-radar/issue/BAK-1189/document-relationships-change-behaviour-to-return-all-relationships